platform/administer/shared-database/shared-database.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Shared database vCluster Platform
+sidebar_label: Shared Database
+sidebar_position: 1
+description: Learn how to use shared database with vcluster.
+---
+
+The shared database feature allows an admin to configure a database server to be used to manage backing stores for multiple virtual clusters. For each virtual cluster, the feature automatically creates individual databases and
+creates a non-privileged user.
+
+
+:::note
+This feature currently only supports MySQL database servers.
+:::
+
+:::note
+This feature _is_ compatible with AWS RDS MySQL clusters.
+:::
+
+
+### Requirements
+
+You must have a running MySQL server and admin credentials to the server.
+
+### Create Platform admin Connector Secret
+
+A connector secret is a secret that can be used with a platform feature or integration.
+
+For platform to start using a database server, a secret must be created that contains admin credentials for the server. The secret must:
+* Contain the `loft.sh/connector-type` label with the `shared-database` value.
+* Reside in the same namespace as vCluster Platform.
+* Contain the fields:
+    * endpoint
+    * password
+    * port
+    * user
+
+The template for a shared database connector secret is:
+```yaml title="Shared database secret template"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <name> # this is a normal metadata name. A descriptive name is recommended. This will be used in a tenet vCluster's config to reference the connector secret
+  namespace: <vcluster-platform-namespace> # this is the namespace that vCluster Platform is installed in. "vcluster-platform" and "loft" are common values.
+  labels:
+    loft.sh/connector-type: "shared-database"
+stringData:
+  endpoint: <endpoint> # this could be a service endoint or an external hostname endpoint
+  password: <password>
+  port: <port> # 3306 is a common port for database servers
+  user: <user>
+
+```
+
+1. Replace all values surrounded by angular brackets with your own value and save to a file as `<filename>`.
+2. Create the secret: `kubectl create -f <filename>`
+
+That's it. A new vCluster can now reference the created connector secret by setting the `controlPlane.backingStore.database.external.connector` field in the vCluster config
```

Comment:
Can we add a vCluster.yaml here instead so it will be easier for users to configure ?

Reply:
Added a full separate doc for vCuster now.